### ISSUE TITLE
A Link to the Past: Fast Rom

### ISSUE THREAD

**Issue by RobG92** [ArchipelagoMW/Archipelago#3185]
### What feature would you like to see?

It would be nice if its possible that the ALttP Rom could be replaced withthe newer Fast Rom. The actuall Slow Rom causes Lags with to many Sprites. Also it doesnt support Boss Icons if you play a Boss Shuffle and found the Compass for the Dungeon.

**Comment by eudaimonistic** [comment]
The features in question are found in a completely different branch of ALTTP,  Due to the diverging basepatch history, it would take significant effort to implement those features, and more importantly _the people who implemented them elsewhere likely do not wish to contribute to this project._  In fact, the hostilities of yesteryear have us being extra cautious about using the work of other branches, even when their licenses permit us to do so without asking.

The simplest truth is that even when we ask to use people's work, we will get accused of stealing.  As a result, any features to be ported over from other ALTTP branches require the original author to do so themselves, or for the feature to be completely rewritten from scratch so there is no space for someone to make those accusations.

In short, these requests are unlikely unless someone wants to do it themselves from scratch.
